### DISCUSSION TITLE
Website Help

### DISCUSSION THREAD

**Discussion by AJBernados** [community/community#78568] [Programming Help]
I wanted an mp4 for my background, but it keeps covering or conflicting with my sidebar. Any suggestions?
This is before the background.
![image](https://github.com/community/community/assets/143686577/3ed8d6e5-6a2b-4369-a920-a8719ba1cb86)
And this is after.
![image](https://github.com/community/community/assets/143686577/575723ec-c8db-410a-a1b5-e038f5d2bcc7)
If you want the HTML & CSS I can show you.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by strxxtz** [comment]
can u show me the html and css for my watch program? please it would be very helpful i put you on dev list

**Reply by AJBernados** [reply]
Why do you need my html and css for a watch program?

**Comment by b0c0de** [comment]
@AJBernados 

1. Get video wallpaper software: PUSH, Wallpaper Engine (Windows), Moom (Mac). Set zones to avoid the sidebar.
2. Adjust sidebar: Make it transparent (Windows) or resize it (Mac).
3. Choose smaller video: 16:9 aspect ratio, zoom or move it to exclude the sidebar.

Enjoy your video background without sidebar clash!

**Comment by strxxtz** [comment]
thank you

**Comment by clyde-dot** [comment]
@AJBernados
It sounds like there might be an issue with the CSS or layout settings in your code. One potential solution could be adjusting the CSS properties for your background and sidebar elements, ensuring they don't overlap or conflict with each other.
Before setting the MP4 as a background, you might want to try the following:
_Z-index: Set a lower z-index for the sidebar compared to the background video. This might ensure that the sidebar stays behind the background video.
Positioning: Use specific positioning like position: fixed or position: absolute for the sidebar and the background video. Adjust their positioning properties (such as top, left, right, bottom) accordingly to avoid overlapping.
Sizing: Ensure the sizes of the background video and the sidebar don’t conflict. Adjust the width and height of the elements as needed._
**Remember, these are general suggestions. Without seeing your code or specific elements, it's challenging to provide precise guidance. If you'd like, you can share snippets of your HTML and CSS code for more targeted advice!**

**Comment by mehrabanbarlou** [comment]
Hello.
You cannot use videos to cover part of the site...
You can use img tag in html and use gif media to cover part of the site.
You can also use flexbox or css grid for overlay.
For example, create a main tag and then create two div elements.
The width of one of them should be 40% and the other should be 60%.
Consider the element that is 40% wide as the sidebar and make changes to it, and select the element that is 60% wide as the element you want to have the image on.
Of course, there is a simpler solution.
It is not necessary to use the img tag in HTML.
Inside the CSS, select the element whose width is 60% and then give it the background image property. Use the url() value. Then give it a gif address.
also you can use object-fit property to make it better.

Set the height of the main element to 100Vh and the height of the other element to 100%.
I think it will make your code better.
